examples/annotation/assessment_prompt.yaml

```
@@ -0,0 +1,42 @@
+
+    Guidelines for verification:
+    - World: International affairs, politics, wars, country-specific events (including economic events of specific countries)
+    - Sports: Athletic competitions, teams, players, sporting events
+    - Business: Corporate earnings, market movements, company performance, general economic indicators
+    - Sci/Tech: Tech Companies, tech news, technology innovations, scientific discoveries, product launches, technical developments
+
+    Now verify this prediction:
+
+    Text: {{text}}
+    Predicted Category: {{output}}
+    Assessment:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)